### ISSUE TITLE
Depreciated Call

### ISSUE THREAD

**Issue by PHPJJ** [DeanCording/node-red-contrib-persist#17]
I get the following error/warning using persist in my flow.

"[RED.events] Deprecated use of "nodes-started" event from "/home/pi/.node-red/node_modules/node-red-contrib-persist/persist.js:164:20". Use "flows:started" instead."

**Comment by StephanStS** [comment]
Should be fixed with PR #12.

**Comment by SirRufo** [comment]
> Should be fixed with PR #12.

No, it does not ... `"nodes-started"` was replaced in PR #12 by `"nodes:started"` instead of `"flows:started"`
